go.mod

```diff
@@ -8,6 +8,7 @@ require (
 	github.com/bgentry/speakeasy v0.1.0
 	github.com/btcsuite/btcd v0.22.0-beta
 	github.com/btcsuite/btcutil v1.0.3-0.20201208143702-a53e38424cce
+	github.com/cockroachdb/apd/v2 v2.0.2
```

Comment:
We should have `x/group` as a separate go module and add all those dependencies in there instead

Reply:
More specifically, we should add the `math` package to `x/group/internal`